tests/ocepa-freq1/input.dat

```diff
@@ -13,15 +13,14 @@ set {
   guess sad
   mo_maxiter 200
 }
-# Test against Analytic second derivatives from PSI3. #TEST
 grad_freqs = psi4.Vector(7)  #TEST
-grad_freqs.set(0, 0, 513.956) #TEST
-grad_freqs.set(0, 1, 513.956) #TEST
-grad_freqs.set(0, 2, 722.308) #TEST
-grad_freqs.set(0, 3, 722.308) #TEST
-grad_freqs.set(0, 4, 1977.857) #TEST
-grad_freqs.set(0, 5, 3406.952) #TEST
-grad_freqs.set(0, 6, 3497.387) #TEST
+grad_freqs.set(0, 0, 513.982) #TEST
+grad_freqs.set(0, 1, 513.982) #TEST
+grad_freqs.set(0, 2, 722.325) #TEST
+grad_freqs.set(0, 3, 722.325) #TEST
+grad_freqs.set(0, 4, 1977.865) #TEST
+grad_freqs.set(0, 5, 3406.975) #TEST
+grad_freqs.set(0, 6, 3497.404) #TEST
```

Comment:
For grads/freqs, it'd be handy to know where new refs are from, whether findif or p4n implementation or just plain the code at a moment when you trust it.

Reply:
All energies and gradients are P4N analytic, while frequencies are by finite difference of P4N analytic gradients. All have <10^-12 residuals on everything, so it's essentially machine precision. I'll update the test inputs to say so tomorrow.

FWIW, my hesitation against doing so was that it's my private P4N implementation, which won't be very useful when people are looking at this two years from now. I'll get said implementations in a P4N-acceptable form and submit them before I graduate, but there are two Very Important Papers that need to be written before I get back to the project that would be for.

Reply:
Thanks for adding notes. The origin is more a breadcrumb guide as to independence of numbers and which university filesys to search if desperate than enough detail to re-run the numbers.

And two more 👍 to this PR for fixing two classes of occ calcs that wouldn't run for me yesterday.

Reply:
To be fair, those `occ` calcs weren't running because I screwed up #1829. I am looking forward to #1841.

Reply:
And, done.